components/Buttons/LibraryPlayButton.vue

```diff
@@ -0,0 +1,64 @@
+<template>
```

Comment:
@MrTimscampi The circular progress is necessary imo because here it takes like 5 seconds to load the 2000 tracks I have

Reply:
There's already a loading indicator for the cards themselves, though, so the play button could just show up?

Or better yet, disable it while stuff isn't loaded.

Reply:
@MrTimscampi No, I mean after clicking on it. They're already disabled while content is loading. The time between pressing the button and starting playback is what it takes more than 5 seconds.

https://user-images.githubusercontent.com/10274099/105629927-2831b500-5e46-11eb-8611-bf857d939ae4.mp4

Reply:
Ah, I hadn't seen that since it's instant for me.

Then the circular progress is fine, but I still think they should be proper buttons on desktop, to fit better with all the rest :yum:

Reply:
@Mrtimscampi Yeah, I'll label this as blocked so your PR goes first